### ISSUE TITLE
Custom version identifier?

### ISSUE THREAD

**Issue by Eeems** [mrsixw/concourse-rsync-resource#16]
How would one go about specifying a custom version identifier during a put?

**Comment by Eeems** [comment]
Looks like @mrsixw hasn't been active on GitHub since 2019.

**Comment by mrsixw** [comment]
how do you mean? Got an example?

**Comment by Eeems** [comment]
Sure, I'm using this to sync package builds for my hosted pacman repo, and instead of using a guid for the version identifier, I'd like to use the package version.

So instead of `/data/concourse/packages-3ds-libccd/9a41b888e1ef32fadb85defe0dcf6886/3ds-libccd-2.1.r0.g7931e76-2-any.pkg.tar.zst` it would have the following final path `/data/concourse/packages-3ds-libccd/2.1.r0.g7931e76-2/3ds-libccd-2.1.r0.g7931e76-2-any.pkg.tar.zst`

**Comment by mrsixw** [comment]
Does `disable_version_path: true` Work?

**Comment by Eeems** [comment]
Not really, I want to be able to trigger further workflows with new versions, as well as have multiple older versions kept around.

**Comment by mrsixw** [comment]
Ah. You might need some sort of extra parameter then, I think the guid is generated by concourse itself. I think, it’s been a while

**Comment by Eeems** [comment]
Looks like it's generated here: https://github.com/mrsixw/concourse-rsync-resource/blob/master/assets/out#L64-L67

**Comment by mrsixw** [comment]
Yeah. So you’d need to modify this to do what you need
